adr/0023-per-provider-reporting.md

```diff
@@ -0,0 +1,28 @@
+# 23. Per-provider Reporting
+
+**Date:** 19/04/2024
+
+## Status:
+
+Proposed
+
+
+## Context
+
+Last cycle we created a static “Mid-cycle Report”, viewable by Provider users in Manage, based on an uploaded CSV file (done only once). See Design History: Letting training providers view their recruitment performance.
+
+This year we are proposing a dynamic per-provider report in Manage of candidate-level statistics, with a metric showing the number of candidates with “no response > 30 days” (i.e. candidates that have an ‘Inactive’ application with this provider).
+We will use the existing format & content as our baseline for an appropriately amended design.
+
+## Proposition:
+
+The previous implementation of the Mid-cycle Report (2022-23) should be removed as the data will be significantly different from this years report - this year has a Candidate focus rather than Application focus.
+As such we have decided to remove the Mid-cycle Report from the UI and purge the data.
+
+The new (Per-provider) report will use a combination of last year's Mid-cycle Report UI blended with data concepts used in the implementation of Monthly Reporting.
```

Comment:
I used the govuk components and guidance for creating complex tables here. I didn't reuse the existing UI at all as it wasn't fit for this purpose. (https://govuk-components.netlify.app/components/table/#complex-tables)

Reply:
Maybe you can just leave this paragraph out, it's not really a helpful level of detail I think.